cirq/contrib/noise_models/noise_models.py

```diff
@@ -321,11 +321,9 @@ def simple_noise_from_calibration_metrics(calibration: engine.Calibration,
     readout_error_probs: Dict['cirq.Qid', float] = {}
 
     if depol_noise:
-        # TODO: replace with Pauli error once it's available.
-        # Github issue: https://github.com/quantumlib/Cirq/issues/2832
         depol_probs = {
             qubit[0]: depol_prob[0] for qubit, depol_prob in
-            calibration['single_qubit_rb_total_error'].items()
+            calibration['single_qubit_rb_pauli_error_per_gate'].items()
```

Comment:
Depolarization probability and Pauli error probability are equal only approximately. The approximation is good for a large number of qubits, but for a single-qubit the two quantities are quite different. See for example equation (42) on p.11 in the [supplement to the quantum supremacy paper](https://arxiv.org/abs/1910.11333). In the single-qubit case, it yields:

```python
depol_prob = sq_pauli_error_prob * 4 / 3
```

Reply:
Thanks for catching this! Added comments and adjusted to reflect this.